Rubberduck.Core/UI/CodeExplorer/Commands/AddComponentCommand.cs

```diff
@@ -1,6 +1,8 @@
-﻿using System.Runtime.InteropServices;
+﻿using System.IO;
+using System.Runtime.InteropServices;
```

Comment:
Why was this added? I might be missing it but I don't see a need for that library in the changes introduced?